src/main/groovy/io/sdkman/service/TextService.groovy

```diff
@@ -34,19 +35,22 @@ class TextService {
 
     String prepare(ArrayList<Broadcast> broadcasts) {
         def output = "$HEADER\n"
-        if(broadcasts)
+        if (broadcasts)
             output += buildMessage(broadcasts)
         else
             output += NO_BROADCASTS
         "${output}$FOOTER"
     }
 
-    private static buildMessage(broadcasts) {
-        def output = ""
-        broadcasts.each { broadcast ->
-            output += "* ${getDateInstance(SHORT, Locale.UK).format(broadcast.date)}: $broadcast.text\n"
-        }
-        output
+    private buildMessage(broadcasts) {
+        broadcasts
+            .collect{"* ${toISO8601(it.date)}: $it.text" + lineSeparator()}
+            .join()
```

Comment:
You can simply do line separation as a parameter to the join function:
```
.join(lineSeparator())
```

Reply:
Unfortunately I'd have to add a + lineSeparator(). The Cuke expects a `lineSeparator()` at the end and the `join()` only adds it between the elements and not at the end. 
The code would look bulkier like: `.join(lineSeparator()) + lineSeparator()` in order for the cukes to pass.

Reply:
I'd prefer it that way tbh.